src/serde.rs

```diff
@@ -100,6 +100,7 @@ where
 {
     let mut deserializer = stry!(Deserializer::from_slice(s.as_bytes_mut()));
 
+    dbg!(&deserializer.tape);
```

Comment:
I assume this is a testing leftover?

Reply:
ah yes! I totally not wanted to put that into every deserialisation :D thank you